src/ai/backend/test/testcases/utils.py

```diff
@@ -0,0 +1,52 @@
+import shutil
+import tempfile
+from pathlib import Path
+from typing import Optional
+
+from ai.backend.client.session import AsyncSession
+
+
+def get_test_temp_dir(test_id: str) -> Path:
+    temp_base = Path(tempfile.gettempdir()) / "backend_ai_tests"
+    temp_dir = temp_base / test_id
+    return temp_dir
+
+
+async def login(
```

Comment:
The login test itself would still be useful.
But we can inject information such as the access key, secret key, and endpoint by passing `APIConfig` as an argument when creating the client session.
My opinion is that injecting APIConfig should be sufficient, as long as the behavior doesn't change when running on top of a login session.
I think managing and storing cookie data in a local path is cumbersome.